packages/block-editor/src/components/contrast-checker/test/__snapshots__/index.js.snap

```diff
@@ -8,24 +8,57 @@ exports[`ContrastChecker should render component when the colors do not meet AA
   isLargeText={true}
   textColor="#666"
 >
-  <div
-    className="editor-contrast-checker block-editor-contrast-checker"
+  <ContrastCheckerMessage
+    backgroundColor="#666"
+    textColor="#666"
+    tinyBackgroundColor={
+      Object {
+        "_a": 1,
+        "_b": 102,
+        "_format": "hex",
+        "_g": 102,
+        "_gradientType": undefined,
+        "_ok": true,
+        "_originalInput": "#666",
+        "_r": 102,
+        "_roundA": 1,
+        "_tc_id": 2,
```

Comment:
These IDs increment with each use, thus can be affected externally and fail the tests.

See: 4c6a9a4 / #18681

Do we have to pass around the TinyColor object? Should we avoid snapshots?

Reply:
yes, these snapshots are not that useful I think.

Reply:
See #19169